migrations/sql/V2019.04.03__create_core_user_tables.sql

```diff
@@ -0,0 +1,63 @@
+CREATE TABLE IF NOT EXISTS core_user
```

Comment:
Is it just GitHub? The formatting seems a bit janky

Reply:
it's line wrapping cause the file has lots of tabs in it, it looks bad on github.